events/omero-users-workshop-dundee-april-2019.html

```diff
@@ -18,7 +18,7 @@ <h2 class="event-day">Location: Dundee</h2>
         </div>
         <div class="row">
             <div class="column">
-                <h2 class="event-day">Date: 08-09/04/2019</h2>
+                <h2 class="event-day">Date: 8th-9th April 2019</h2>
```

Comment:
https://snoopycrimecop.github.io/www.openmicroscopy.org/events/ gives `April 08-09, 2019`. Worth unifying the date format ?

Reply:
I don't think so: the other events aren't unified and the context is different. (The index is a tiny note next to the calendar icon, this page is rather more verbose about the event.)